test/tests/ver-1d/ver-1d-trapping.i

```diff
@@ -88,8 +94,8 @@ temperature = 1000
   [trapping]
     type = TrappingNodalKernel
     variable = trapped
-    alpha_t = ${fparse 1e15 / time_scaling}
-    N = ${fparse 3.1622e22 / cl}
+    alpha_t = '${fparse 1e15 / time_scaling}'
+    N = '${fparse ${N} / cl}'
```

Comment:
```suggestion
    N = '${fparse N / cl}'
```
I don't think you need `${}` here since you are already in `${fparse }`.

Reply:
Removed.